agent/provider/src/startup_config.rs

```diff
@@ -142,7 +143,61 @@ impl ProviderConfig {
     }
 }
 
-#[derive(StructOpt, Clone, Debug, Serialize, Deserialize, derive_more::Display)]
+#[derive(Clone, Debug)]
+pub struct NetworkOrPlatform {
+    pub driver: String,
```

Comment:
I'm wondering if we have typed structure somewhere?

Reply:
No*, but I'll replace this with a newtype around the `PaymentPlatform` when I'm done with the refactor.

*actually yes, but it's driver-specific implementation detail, and I intend to remove it as well